src/components/Payment/ChargeInfo.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import styled from "styled-components";
+
+const ChargeInfo = () => {
+  return (
+    <PaymentInfo>
+      <PaymentInfoTitle>요금 세부정보</PaymentInfoTitle>
+      <ChargeFee>
+        <PaymentDetail>
+          <Item>20,000원 x 2시간</Item>
+          <Rate>40,000원</Rate>
```

Comment:
```suggestion
          <div>20,000원 x 2시간</div>
          <div>40,000원</div>
```
아래 리뷰대로라면 이렇게 적용하시면 되겠습니다!